File: ai-service/src/main/java/uvt/orar/dto/TeacherDTO.java
package uvt.orar.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class TeacherDTO {
    private UUID id;
    private String title;
    private String firstName;
    private String lastName;
    private String email;
    private String phone;
}

File: generator-service/app/models/TeacherDTO.py
from uuid import UUID

from pydantic import BaseModel, ConfigDict


class TeacherDTO(BaseModel):
    id: UUID | None = None
    title: str | None = None
    firstName: str | None = None
    lastName: str | None = None
    email: str | None = None
    phone: str | None = None
    intern: bool | None = None

    model_config = ConfigDict(from_attributes=True)
